.github/CODEOWNERS

```
@@ -0,0 +1,2 @@
+# Team
+* @andyduong1920 @hoangmir @junan @longnd @nvminhtue
```

Please correct my name 🙏 
```suggestion
* @andyduong1920 @hoangmirs @junan @longnd @nvminhtue
```

---

Oh sorry my bad, I have fixed it in https://github.com/nimblehq/nimble-growth-39-k8s-aws-eks/pull/11/commits/b9bbc7d7b24240d5064d3112d6be84f575503143